src/freenet/clients/fcp/DirPutFile.java

```diff
@@ -50,7 +50,7 @@ public static DirPutFile create(SimpleFieldSet subset, String identifier, boolea
 		if(name == null)
 			throw new MessageInvalidException(ProtocolErrorMessage.MISSING_FIELD, "Missing field Name", identifier, global);
 		String contentTypeOverride = subset.get("Metadata.ContentType");
-		if(contentTypeOverride != null && (!contentTypeOverride.equals("")) && !DefaultMIMETypes.isPlausibleMIMEType(contentTypeOverride)) {
+		if(contentTypeOverride != null && (!contentTypeOverride.isEmpty()) && !DefaultMIMETypes.isPlausibleMIMEType(contentTypeOverride)) {
```

Comment:
The parentheses around the `isEmpty()` are unnecessary.

Reply:
Fixed in [a29fa9d].
Thank you